src/Catty/ViewController/Help/HelpWebViewController.swift

```diff
@@ -314,7 +352,9 @@ class HelpWebViewController: UIViewController, UIWebViewDelegate, UIScrollViewDe
         hud?.center = CGPoint(x: view.center.x,
                               y: view.center.y + CGFloat(kBDKNotifyHUDCenterOffsetY))
         hud?.tag = Int(kSavedViewTag)
-        view.addSubview(hud!)
+        if hud != nil {
+            view.addSubview(hud!)
```

Comment:
This will still produce a linter warnings because of force unwrapping. Use `if let` instead.